src/components/TableModule/TableModule.tsx

```diff
@@ -374,8 +378,9 @@ export const TableModule = React.memo(
           });
           setStickyCellsLeft(stickyCellsLeft);
         } else {
-          console.warn(
-            "Table's forwardRef is null, please set it if you want the sticky cell's left value to be set correctly"
+          warning(
+            process.env.NODE_ENV === 'development',
+            "Chroma Warning: Table's forwardRef is null, please set it in order to set the sticky cell's left value correctly"
```

Comment:
`process` object is only available in node.js runtime. it will report error when interpreting it in a browser. So it could just keep these lines as they were

Reply:
OK - I was modeling this warning off the one on line 291 - should I adjust that to a plain console.warn as well? I think this util is common around Chrōma, so I'm wondering if we have a larger issue to address.

Reply:
just use `console.warn()` like what it was. I don't think you need to change these lines except your fix

Reply:
(I also agree with Shawn's immediate feedback: I'd suggest skipping the `process.env` check and just doing `console.warn`)

Reply:
Cool, thanks for the feedback, guys!